tutorials/asr/Lhotse_Dataloading.ipynb

```diff
@@ -0,0 +1,459 @@
+    "- A specific method of blending multiple datasets together, based on a probabilistic multiplexer: [CutSet.mux](https://lhotse.readthedocs.io/en/latest/api.html?highlight=mux#lhotse.cut.CutSet.mux).\n",
+    "\n",
+    "### How is Lhotse integrated into NeMo?\n",
+    "\n",
+    "Like NeMo, Lhotse leverages JSON Lines (JSONL) format to keep metadata in manifests. Unlike NeMo, Lhotse's manifests are mapped into Python objects. Most data represented by NeMo manifests are a special case supported by Lhotse and can be adapted on-the-fly to be directly read as a Lhotse `CutSet`. This conversion is enabled by [LazyNeMoIterator](https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/common/data/lhotse/nemo_adapters.py#L33) and [LazyTarredNeMoIterator](https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/common/data/lhotse/nemo_adapters.py#L131) classes inside NeMo.\n",
+    "\n",
+    "Building a `DataLoader` with Lhotse relies heavily on `CutSampler` to sample a mini-batch on metadata level as a `CutSet`, which is then passed to a map-style PyTorch dataset that converts the `CutSet` to a tuple/dict of tensors. This is covered in more detail in Lhotse tutorials:\n",
+    "- Introductory tutorial [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/lhotse-speech/lhotse/blob/master/examples/00-basic-workflow.ipynb)\n",
+    "- Tutorial on Lhotse Shar format (\"tarred\" data format in Lhotse): [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/lhotse-speech/lhotse/blob/master/examples/04-lhotse-shar.ipynb)\n",
+    "\n",
+    "The dataloader is being constructed in NeMo from a dataset configuration using function [get_lhotse_dataloader_from_config](https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/common/data/lhotse/dataloader.py#L103).\n",
+    "\n",
+    "### Getting mini LibriSpeech\n",
+    "\n",
+    "For the sake of the tutorial, we'll be using very small datasets to keep things quick. The first example is mini LibriSpeech, a 5h train set and 2h dev set collection that was released alongside the popular LibriSpeech dataset. Lhotse provides a download function that fetches and unpacks the archive with data, and a prepare function that creates Lhotse manifests."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "93f4033f-ec05-4ead-9675-72dc3d36a772",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "libri_root = download_librispeech(root_dir, dataset_parts=\"mini_librispeech\")\n",
+    "\n",
+    "libri = prepare_librispeech(\n",
+    "    libri_root, output_dir=root_dir, num_jobs=num_jobs\n",
+    ")\n",
+    "\n",
+    "for split in (\"train-clean-5\", \"dev-clean-2\"):\n",
+    "    (\n",
+    "        CutSet\n",
+    "        .from_manifests(**libri[split])\n",
```

Comment:
it's not obvious why `**libri[split]` works, is there a simple way to explain it, maybe in the code comments?
maybe doesn't matter too much if it's just a "how-to"